tables/AASYSJUR.sql

```diff
@@ -1,44 +1,44 @@
 CREATE TABLE `iasworld.aasysjur`(
-  `jur` varchar(6), 
```

Comment:
In cases where the table was never bucketed, there will be line-level changes like this. In the bucketed case, an additional version of the table is removed.